report_thesis/src/sections/related_work.tex

```diff
@@ -46,11 +49,78 @@ \section{Related Work}
+The study finds that while submodels can improve prediction accuracies under certain conditions, the overall effectiveness is contingent upon having a large and diverse calibration dataset.
+The research suggests that the optimal use of \gls{libs} for geochemical analysis requires a balance between the specificity of submodels and the breadth of the calibration dataset to ensure accurate and reliable predictions.
+
+\citet{kepesImprovingLaserinducedBreakdown2022} discusses enhancing \gls{libs} model accuracy using transfer learning between ChemCam and SuperCam instruments.
+It proposes a method where ChemCam data transforms to approximate SuperCam spectra, improving \gls{cnn} regression models' performance.
+Key methods include data augmentation and fine-tuning of \gls{cnn}s with pre-processed and normalized spectra.
+This approach outperforms some existing models for specific oxides, demonstrating transfer learning's potential in \gls{libs} analyses for more accurate quantitative models.
+
+\citet{ferreiraComprehensiveComparisonLinear2022} presents an extensive comparison of various algorithms for quantifying lithium in geological samples, with a focus on both linear and non-linear methods.
+The study tested algorithms on spectra acquired from a commercial handheld device and a laboratory prototype, highlighting the challenges in quantifying lithium due to effects like saturation and matrix interference.
+The results showed that non-linear methodologies, such as K-nearest neighbors regression, Support Vector Machine regression, and Artificial Neural Networks regression, generally outperformed linear methods by effectively managing saturation and matrix effects, which are common in geological samples.
+This research provides valuable insights for future applications in geological sample analysis and could potentially be generalized for other elements in similar contexts.
+The paper's findings are particularly relevant to our project as it demonstrates the effectiveness of non-linear machine learning techniques in handling complex, non-linear relationships in high-dimensional \gls{libs} data, aligning with our research objectives of improving major oxide composition predictions from \gls{libs} data.
+
+The study by \citet{liuComparisonQuantitativeAnalysis2022} explores the use of MarSCoDe \gls{libs} for quantitative analysis of olivine in a simulated Martian atmosphere, focusing on multivariate analysis methods to address challenges posed by \gls{libs} data, such as high dimensionality and multicollinearity.
+The methods evaluated include univariate linear regression (ULR), multivariate linear regression (MLR), principal component regression (PCR), partial least squares regression (PLSR), ridge regression, \gls{lasso}, \gls{enet}, and back-propagation neural networks.
```

Comment:
ULR, MLR, PCR and PLSR in glossary (also back-propagation neural networks should already be in glossary so use that).